### ISSUE TITLE
Data Validation and ETL" lab - wrong Compression Type value

### ISSUE THREAD

**Issue by benoitdechateauvieux** [aws-samples/data-engineering-for-aws-immersion-day#20]
Hi,
In the "Data Validation and ETL" lab, when transforming to Parquet, if the default value for `Compression Type` (`None`), is used in the "Data targets", an error is raised and the job fails:
```
An error occurred while calling o101.pyWriteDynamicFrame. No enum constant org.apache.parquet.hadoop.metadata.CompressionCodecName.NONE
```
When the `Compression Type` is changed to `Uncompressed`, the jobs run fine.

**Comment by ghost** [comment]
Fix is available as part of PR #22

**Comment by melodyyangaws** [comment]
PR is merged. close the issue.
